carbon-design-system/carbon-addons-iot-react
# fix(list): adjust hierarchy indentations to match spec

Closes #1568

**Summary**

- HierarchyList indentation is improved for when there's a mix of category and non-category items.

![image](https://user-images.githubusercontent.com/3360588/93821956-08cf1500-fc25-11ea-95fe-e6fad359c84c.png)



**Change List (commits, features, bugs, etc)**

- Root level items with no children align with items that have children
- Children items have slightly more indentation than their parent
- Flipped the caret direction to be consistent with sidenav and accordion. When the section is closed, the arrow points down.

**Acceptance Test (how to verify the PR)**

- View and validate the new [`HierarchyList \ with mixed hierarchies`](https://deploy-preview-1617--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-experimental-hierarchylist--with-mixed-hierarchies) story
- Check the other HierarchyList stories to ensure no visual regressions
